Open_Foam/Post_Process.ipynb

```diff
@@ -4,129 +4,112 @@
-       "        "
-      ],
-      "text/plain": [
-       "<IPython.lib.display.YouTubeVideo at 0x1041d0f98>"
-      ]
-     },
-     "execution_count": 1,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "from IPython.display import YouTubeVideo\n",
-    "YouTubeVideo(\"kyAJiOXQIq4\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Write an Equation\n",
     "\n",
-    "### Within Text Body\n",
-    "Use single `$ <equation> $` symbol to put and equation (e.g. ```$y = \\delta ~,~ u = 0.99 ~U_e .$```) within the text body.\n",
+    "Execution of the `vorticity` command will calculate and save vorticity field data for all time steps. Now again by running `paraFoam` and loading the variables, including the calculated vorticity data, user can choose the vorticity field from the drop down list and visualize it within the CFD domain. Fig.3 shows vorticity contours around the cylinder for TSR of 0.5 and 2.\n",
     "\n",
-    "Use single `$ <equation> $` symbol to put and equation (e.g. $y = \\delta ~,~ u = 0.99 ~U_e .$) within the text body.\n",
+    "<img src=\"./Images/3vortContours.png\" width=\"800\" align=\"middle\">\n",
+    "</br>\n",
+    "Fig. 3 - Vorticity contours for flow over a rotating cylinder with Re=100 and a) $TSR=0.5$ and b) $TSR=2$ at timestep of `t=150`.\n",
     "\n",
-    "### Standalone\n",
-    "```\n",
-    "Use double `$$ <equation> $$` symbol to put equation in standalone format in the center of the page:\n",
-    "\n",
-    "$$\n",
-    "\\mathbf{R_{ext}} + \\mathbf{F_v} -\n",
-    "\\int \\mathbf{n}~p~dA +\n",
-    "\\int \\rho \\mathbf{g}~dV = \n",
-    "\\frac{\\partial}{\\partial t} \\int \\rho \\mathbf{V} dV +\n",
-    "\\int (\\mathbf{n}~.~\\rho \\mathbf{V}) \\mathbf{V} dA .\n",
-    "$$\n",
-    "```\n",
+    "Suppression of the vortex shedding in the wake of the rotating cylinder removes the previously existing pressure gradient across the cylinder, due to pre-existing low-pressure region just behind the cylinder. As a result of this phenomena a lower drag forceis experienced by the cylinder compared to the case study of stationary cylinder. On the other other side of the spectrum, rotation of the cylinder leads to a velocity gradient, hence a pressure gradient, between the top and bottom of the cylinder as visualized in Fig. 2(b). This gradient will lead to a force acting normal to streamwise flow direction and generates a \"lift force\". This is an entirely new phenomena, compared to the stationary cylinder, due to the rotation of the cylinder and is called the **Magnus Effect** . It should be recalled that in the case of stationary cylinder, due to the flow field symmetry, the normal component of the forces exerted on the cylinder's wall would cancel each other out and lift force will be almost zero. For more information on the topic of **Magnus Effect** please see [here](https://en.wikipedia.org/wiki/Magnus_effect).\n",
```

Comment:
The content and explanation of this paragraph went under major revision. Please double check the accuracy and validity it. Either approve it or implement more modifications.